### ISSUE TITLE
emoj not working if you are not running it within the folder with package.json

### ISSUE THREAD

**Issue by MaciejTrojniarz** [sindresorhus/emoj#44]
I've tried to run emoj in the terminal but for some reason it requires projectName option and doesn't work from outside of the catalog of the any project with package.json.

```bash
npx emoj 'Not working when not in JS project'
Project name could not be inferred. Please specify the `projectName` option.
```

**Comment by sindresorhus** [comment]
Fixed in 3.0.1

**Comment by MaciejTrojniarz** [comment]
That was fast. I've just finished fix I wanted to create pull request. Thanks @sindresorhus
